packages/flutter_goldens/pubspec.yaml

```diff
@@ -8,30 +8,30 @@ dependencies:
   # To update these, use "flutter update-packages --force-upgrade".
```

Comment:
this package itself seems to have sdk constraint that needs to be adjusted: https://github.com/flutter/flutter/blob/master/packages/flutter_goldens/pubspec.yaml#L5

Reply:
I could do that as a part of this PR as well, should I?

Reply:
I see also this one which is totally invalid... https://github.com/flutter/flutter/blob/master/packages/flutter_goldens_client/pubspec.yaml#L5. I think these might just be ignored since they aren't pub packages.

Reply:
as well as flutter_goldens_client https://github.com/flutter/flutter/blob/master/packages/flutter_goldens_client/pubspec.yaml#L5

Reply:
> I could do that as a part of this PR as well, should I?

Yeah, I think so

Reply:
I will go ahead and just update them.